client/apps/admin-portal/src/app/sermon-view/sermon-view.component.ts

```diff
@@ -45,21 +49,8 @@ export class SermonViewComponent implements OnInit {
     this.tagsFacade.loadTagsBySermonId(currentSermonId);
   }
 
-  handleMediaAction(media: Media) {
-    switch(media.type) {
-      case MediaTypes.AUDIO: {
-        return this.listenToAudio();
-      }
-      case MediaTypes.NOTES: {
-        return this.downloadNotes();
-      }
-      case MediaTypes.VIDEO: {
-        return this.downloadVideo();
-      }
-      default: {
-        break;
-      }
-    }
+  grabSermon(currentSermonId: string) {
```

Comment:
I don't think we need this method either since we are already "grabbing" the sermon in `ngOnInit`